Travis failure on pypy with latest package levels

The manual-ci-run fails for pypy and PACKAGE_LEVEL=latest.

Failing Travis run: https://travis-ci.org/zhmcclient/zhmc-ansible-modules/jobs/468325752

Relevant section:
```
bash -c 'PYTHONPATH=../ansible/lib ../ansible/test/sanity/validate-modules/validate-modules zhmc_ansible_modules/zhmc_crypto_attachment.py zhmc_ansible_modules/zhmc_hba.py zhmc_ansible_modules/zhmc_nic.py zhmc_ansible_modules/zhmc_partition.py zhmc_ansible_modules/zhmc_storage_group_attachment.py zhmc_ansible_modules/zhmc_storage_group.py zhmc_ansible_modules/zhmc_storage_volume.py zhmc_ansible_modules/zhmc_virtual_function.py 2>&1 |grep -v -E "(E101|E105|E106)" |tee validate.log.tmp'
/home/travis/virtualenv/pypy-5.3.1/site-packages/cryptography/hazmat/bindings/openssl/binding.py:163: CryptographyDeprecationWarning: OpenSSL version 1.0.1 is no longer supported by the OpenSSL project, please upgrade. A future version of cryptography will drop support for it.
  utils.CryptographyDeprecationWarning
bash -c 'if [[ -n "$(cat validate.log.tmp)" ]]; then false; fi'
make: *** [validate.log] Error 1
The command "make check" exited with 2.
```

The root cause is that OpenSSL is at version 1.0.1 in that environment, causing the "cryptography" package to issue a `CryptographyDeprecationWarning` in the Ansible validate-modules check.

--- a/Makefile
+++ b/Makefile
@@ -171,7 +171,7 @@ plugin_formatter_template_dir := $(shell dirname $(plugin_formatter_template_fil
 # validate-modules tool from Ansible project
 validate_modules := $(ansible_repo_dir)/test/sanity/validate-modules/validate-modules
 validate_modules_log_file := validate.log
-validate_modules_exclude_pattern := (E101|E105|E106)
+validate_modules_exclude_pattern := (E101|E105|E106|DeprecationWarning)
 
 # Documentation files from Ansible repo to copy to $(doc_gen_dir)
 doc_copy_files := \
@@ -261,7 +261,7 @@ uninstall:
 .PHONY: clobber
 clobber:
 	rm -Rf .cache $(package_name_pypi_under).egg-info .eggs $(build_dir) $(doc_check_dir) htmlcov .tox
-	rm -f MANIFEST MANIFEST.in AUTHORS ChangeLog .coverage flake8_*.log test_*.log validate.log
+	rm -f MANIFEST MANIFEST.in AUTHORS ChangeLog .coverage flake8_*.log test_*.log $(validate_modules_log_file)
 	find . -name "*.pyc" -delete -o -name "__pycache__" -delete -o -name "*.tmp" -delete -o -name "tmp_*" -delete
 	@echo 'Done: Removed all build products to get to a fresh state.'
 	@echo '$@ done.'

--- a/docs/changes.rst
+++ b/docs/changes.rst
@@ -42,6 +42,11 @@ Released: not yet
   input parameter, and not for the `match` input parameter. The type conversions
   are now applied for all properties of Adapter also for the `match` parameter.
 
+* The dictionary to check input properties for the `zhmc_cpc` module had the
+  `acceptable_status` property written with a hyphen instead of underscore.
+  This had the effect that it was rejected as non-writeable when specifying
+  it as input.
+
 **Enhancements:**
 
 * Docs: Improved and fixed the documentation how to release a version

--- a/zhmc_ansible_modules/zhmc_cpc.py
+++ b/zhmc_ansible_modules/zhmc_cpc.py
@@ -200,7 +200,7 @@ ZHMC_CPC_PROPERTIES = {
 
     # update properties:
     'description': (True, None, True, True, None, to_unicode),
-    'acceptable-status': (True, None, True, True, None, None),
+    'acceptable_status': (True, None, True, True, None, None),
 
     # read-only properties (subset):
     'name': (False, None, False, None, None, None),  # provided in 'name' parm

